tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataPage.java

```diff
@@ -1419,10 +1421,24 @@ public static CheckCLDR.Options getOptions(CookieSession session, CLDRLocale loc
 
         final String org = session.getEffectiveCoverageLevel(locale.toString());
 
-        options = new Options(locale, SurveyMain.getTestPhase(), def, org);
+        options = getOptions(locale, def, org);
         return options;
     }
 
+    private static CheckCLDR.Options getOptions(
+            CLDRLocale locale, final String defaultLevel, final String org) {
+        return new Options(locale, SurveyMain.getTestPhase(), defaultLevel, org);
+    }
+
+    /** Get options, but don't try to check user preferences */
+    public static CheckCLDR.Options getSimpleOptions(CLDRLocale locale) {
+        return new Options(
+                locale,
+                SurveyMain.getTestPhase(),
+                Level.COMPREHENSIVE.name(), /* localeType is not used?! */
+                "NOT USED");
```

Comment:
CoverageLevel_localeType has "set" but no "get", and "localeType" doesn't occur in any jsp files

cruft? delete?

Reply:
"don't use user settings for voting/getrow queries" -- how do we know the user settings aren't accessed?

for that matter, how do we know whether the user settings are accessed when getOptions is used instead of getSimpleOptions?

Reply:
> cruft

seems like it.  How does CheckCLDR use the 'level' setting? Maybe @macchiati can comment on it.

> how do we know the user settings aren't accessed

This PR stops calling into user settings for GET/POST on a row, and uses `COMPREHENSIVE` instead of checking the level.

> for that matter, how do we know whether the user settings are accessed when getOptions is used instead of getSimpleOptions?

`getListSetting()` calls out to the DB to check the user's setting.

```
    public static CheckCLDR.Options getOptions(CookieSession session, CLDRLocale locale) {
        CheckCLDR.Options options;
        final String def =
                CookieSession.sm.getListSetting(
                        session.settings(), SurveyMain.PREF_COVLEV, WebContext.PREF_COVLEV_LIST);

        final String org = session.getEffectiveCoverageLevel(locale.toString());
```

Reply:
> CoverageLevel_localeType has "set" but no "get", and "localeType" doesn't occur in any jsp files
> 
> cruft? delete?

Want to split that out to a separate PR from the performance stuff?